asyncapi/asb/service_types.bal

```diff
@@ -1,4 +1,4 @@
-# The Azure service bus messaging service type.
+# Actions related to Azure service bus messaging. Available method is onMessage
```

Comment:
do we refer to them as methods or actions? 

Rather than saying "Actions related to Azure service bus messaging" can we say "Triggers when a new message is received from Azure service bus". I think the latter gives us a better idea on what this Webhook does. 

Also, shall we use a common format to introduce the actions? Maybe "Available method(s): <methods>"